cli/run_trigger_params.go

```diff
@@ -44,7 +45,7 @@ func parseRunAndTriggerJSONParams(jsonParams string) (RunAndTriggerParamsModel,
 func parseRunAndTriggerParams(
 	workflowToRunID,
 	triggerPattern,
-	pushBranch, prSourceBranch, prTargetBranch, tag,
+	pushBranch, prSourceBranch, prTargetBranch string, isDraftPR *bool, tag,
```

Comment:
I think the idea was that the CLI flag values are passed into this function as input variables like workflowToRunID, triggerPattern, pushBranch, ... But you can pass the whole RunAndTriggerParamsModel as a JSON value (or a base64 encoded JSON value) and flag values should override the values in the JSON params model, when provided.

This was partially true because you can't clear JSON values, by passing an empty string for the relevant flag value. (But you can override by passing non empty strings).

So I tried to follow this intention: if isDraftPR is not nil, then it overrides jsonParams.IsDraftPR.